### PR TITLE
tflint 0.10.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/tflint.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Tflint < Formula
   desc "Linter for Terraform files"
   homepage "https://github.com/wata727/tflint"
-  url "https://github.com/wata727/tflint/archive/v0.9.3.tar.gz"
-  sha256 "cad7d122178175eff12dad349225b4feeb02e34e0f5c021d0d73cb694b04ceb2"
-  head "https://github.com/wata727/tflint.git"
```
Comment (lembacon) [inline Formula/tflint.rb]: Should we keep the `head`?

Comment (chenrui333) [reply Formula/tflint.rb]: I dont think so. We dont use `head` in the formula.

Comment (lembacon) [reply Formula/tflint.rb]: If the HEAD build does not break, we probably should not remove it, even though we generally do not want to add `head` for new formulae.

Comment (chenrui333) [reply Formula/tflint.rb]: Ok. I will add it back then.

Comment (chenrui333) [reply Formula/tflint.rb]: Updated. :)

Comment (lembacon) [reply Formula/tflint.rb]: Thanks!

**Pull request closed without merging**